libfwupdplugin/fu-archive-firmware.c

```diff
@@ -46,14 +51,45 @@ fu_archive_firmware_parse(FuFirmware *firmware,
 	g_autoptr(FuArchive) archive = NULL;
 
 	/* load archive */
-	archive = fu_archive_new(fw, FU_ARCHIVE_FLAG_IGNORE_PATH, error);
```

Comment:
I will revert the renaming in the next iteration.

Reply:
Ya, I think fu_archive_new() is ABI -- although we could change `fw` to be NULL if you want to create an empty archive...